### PR TITLE
feature(gemini_cdc): Add gemini write test for cdc

### PR DESCRIPTION
**Pull request by aleksbykov**
Added new feature CDC tests with gemini.

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I gave variables/functions meaningful self-explanatory names
- [ ] I didn't leave commented-out/debugging code
- [ ] I didn't copy-paste code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by aleksbykov** [comment]
Staging job failed: with scylla issue https://github.com/scylladb/scylla/issues/5920

File: test-cases/gemini/gemini-3h-cdc-write.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+instance_type_db: 'i3.large'
+
+store_results_in_elasticsearch: False
+user_prefix: "gemini-cdc-write"
+
+gemini_cmd: "gemini -d --duration 3h \
+-c 50 -m write -f --non-interactive \
+--cql-features all \
+--replication-strategy \"{'class': 'SimpleStrategy', 'replication_factor': '3'}\" \
+--table-options \"cdc={'enabled': true}\""
```
Comment (bentsi) [inline test-cases/gemini/gemini-3h-cdc-write.yaml]: is the `'preimage': false` here?

Comment (amoskong) [reply test-cases/gemini/gemini-3h-cdc-write.yaml]: false is the default value.

Comment (bentsi) [reply test-cases/gemini/gemini-3h-cdc-write.yaml]: worth to mention explicitly

Comment (aleksbykov) [reply test-cases/gemini/gemini-3h-cdc-write.yaml]: You think, better to set in config_params?

Comment (bentsi) [reply test-cases/gemini/gemini-3h-cdc-write.yaml]: yes

Comment (aleksbykov) [reply test-cases/gemini/gemini-3h-cdc-write.yaml]: Fixed

**Comment by roydahan** [comment]
@bentsi  & @amoskong please review

File: test-cases/gemini/gemini-3h-cdc-write.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+store_results_in_elasticsearch: False
+user_prefix: "gemini-cdc-write"
+
+gemini_cmd: "gemini -d --duration 3h \
+-c 50 -m write -f --non-interactive \
+--cql-features all \
+--replication-strategy \"{'class': 'SimpleStrategy', 'replication_factor': '3'}\" "
+
+gemini_table_options:
+  - "cdc={'enabled': true}"
```
Comment (amoskong) [inline test-cases/gemini/gemini-3h-cdc-write.yaml]: Can we enable preimage, postimage? are they supported by C*?

File: test-cases/gemini/gemini-3h-cdc-preimage-write.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
+test_duration: 300
+n_db_nodes: 3
+n_loaders: 1
+n_monitor_nodes: 1
+instance_type_db: 'i3.large'
+
+store_results_in_elasticsearch: False
+user_prefix: "gemini-cdc-preimage-write"
```
Comment (amoskong) [inline test-cases/gemini/gemini-3h-cdc-preimage-write.yaml]: preimage isn't enabled in gemini_table_options

File: test-cases/gemini/gemini-3h-cdc-write.yaml (hunk 2 above)
Comment (aleksbykov) [reply test-cases/gemini/gemini-3h-cdc-write.yaml]: Yes it should be enabled in gemini_table_options config parameter. Missed it in preimage.
Fixed

File: test-cases/gemini/gemini-3h-cdc-preimage-write.yaml (hunk 3 above)
Comment (aleksbykov) [reply test-cases/gemini/gemini-3h-cdc-preimage-write.yaml]: Fixed

**Pull request merged**